surface/include/pcl/surface/impl/mls.hpp

```diff
@@ -351,35 +306,46 @@ pcl::MovingLeastSquaresOMP<PointInT, PointOutT>::performProcessing (PointCloudOu
     std::vector<float> nn_sqr_dists;
 
     // Get the initial estimates of point positions and their neighborhoods
-    if (this->searchForNeighbors ((*indices_)[cp], nn_indices, nn_sqr_dists))
+    if (searchForNeighbors ((*indices_)[cp], nn_indices, nn_sqr_dists))
     {
-      // Check the number of nearest neighbors for normal estimation (and later
-      // for polynomial fit as well)
+      // Check the number of nearest neighbors for normal estimation (and later for polynomial fit as well)
       if (nn_indices.size () >= 3)
       {
         // This thread's ID (range 0 to threads-1)
-        int tn = omp_get_thread_num ();
-
+#ifdef _OPENMP
+        const int tn = omp_get_thread_num ();
         // Size of projected points before computeMLSPointNormal () adds points
         size_t pp_size = projected_points[tn].size ();
+#else
+        PointCloudOut projected_points;
+        NormalCloud projected_points_normals;
+#endif
 
         // Get a plane approximating the local surface's tangent and project point onto it
-        int index = (*indices_)[cp];
-        size_t mls_result_index = 0;
-        
-        if (this->cache_mls_results_)
+        const int index = (*indices_)[cp];
+
+        if (cache_mls_results_)
           mls_result_index = index; // otherwise we give it a dummy location.
-        
-        this->computeMLSPointNormal (index, nn_indices, projected_points[tn], projected_points_normals[tn], corresponding_input_indices[tn], this->mls_results_[mls_result_index]);
+
+#ifdef _OPENMP
+        computeMLSPointNormal (index, nn_indices, projected_points[tn], projected_points_normals[tn], corresponding_input_indices[tn], mls_results_[mls_result_index]);
 
         // Copy all information from the input cloud to the output points (not doing any interpolation)
         for (size_t pp = pp_size; pp < projected_points[tn].size (); ++pp)
-          this->copyMissingFields (input_->points[(*indices_)[cp]], projected_points[tn][pp]);
-	    }
-	  }
+          copyMissingFields (input_->points[(*indices_)[cp]], projected_points[tn][pp]);
+#else
+        computeMLSPointNormal (index, nn_indices, projected_points, projected_points_normals, *corresponding_input_indices_, mls_results_[mls_result_index]);
+    
+        // Append projected points to output
+        output.insert(output.end (), projected_points.begin (), projected_points.end ());
```

Comment:
There were more missing spaces.
I have updated the uncrustify config so that the indentation of the access qualifiers matches the PCL style guide and formatted the mls-files in commit 1032788148f1c26731a0475d796689917dcb452f.

[Uncrustify_PCL.cfg.txt](https://github.com/PointCloudLibrary/pcl/files/2097202/Uncrustify_PCL.cfg.txt)